docs/Tutorials/CheckpointRestart.md

```diff
@@ -31,9 +31,9 @@ Executables can checkpoint when:
 
 To restart an executable from a checkpoint file, run a command like this:
 ```
-./MySpectreExecutable +restart SpectreCheckpoint000123
+./MySpectreExecutable +restart Checkpoints/Checkpoint123
 ```
-where the `000123` should be the number of the checkpoint to restart from.
+where the `123` should be the number of the checkpoint to restart from.
```

Comment:
Whatever we decide in the discussion on formatting, this documentation should make it clear that these are zero-padded, either by stating it or by using an example with a leading zero.